libs/wire-subsystems/test/unit/Wire/ActivationCodeStore/InterpreterSpec.hs

```diff
@@ -0,0 +1,34 @@
+module Wire.ActivationCodeStore.InterpreterSpec (spec) where
+
+import Data.Default
+import Data.Map qualified as Map
+import Imports
+import Test.Hspec
+import Test.Hspec.QuickCheck
+import Test.QuickCheck
+import Wire.API.User.Activation
+import Wire.ActivationCodeStore
+import Wire.MiniBackend
+import Wire.MockInterpreters.ActivationCodeStore
+
+spec :: Spec
+spec = do
+  describe "ActivationCodeStore effect" $ do
```

Comment:
Have you considered to add a test where the lookup fails? E.g. there is a code, but the key does not match?

Reply:
Just added it! I've rebased and pushed.

Reply:
Cool, thanks! (The more test code coverage we can add, the better - as long as the task doesn't grow out of bounds, of course :smile: )